Tutorials/HowToAddControlAlgorithms.md

```diff
@@ -24,6 +24,7 @@
 - To measure physical quantities, users can use getter functions defined in the `Attitude`, `Orbit`, and `LocalEnvironment` classes as `dynamics_->GetAttitude().GetOmega_b()`.
 - To generate torque and force, users can use `dynamics_->AddTorque_b` and `dynamics_->AddForce_b`.
 - The sample codes are in `SampleCodes/ControlAlgorithm/DirectMethod/UserSat.cpp`, and you can see very simple detumbling with the proportional control method.
+  - If you can't find `virtual void UserSat::Update(const SimTime* sim_time);` in `UserSat.hpp`, add this statement as the public member function.
```

Comment:
Could you add the statement in the sample code [here](https://github.com/ut-issl/s2e-documents/blob/develop/Tutorials/SampleCodes/s2e-user/src/Simulation/Spacecraft/UserSat.hpp) instead of this modification?  
And `virtual` is not necessary so please remove it.

Reply:
I've dealt with it.